dropzonejs.libraries.yml

```diff
@@ -0,0 +1,10 @@
+dropzonejs:
```

Comment:
I am not sure about how to deal with the vendor stuff as there is no good way to use composer with js stuff. For now I've put it in /assets/dropzonejs. I was also thinking /vendor/dropzonejs